docs/website/docs/walkthroughs/create-a-pipeline.md

```diff
@@ -2,4 +2,142 @@
 sidebar_position: 1
 ---
 
-# Create a pipeline
\ No newline at end of file
+# Create a pipeline
+
+Follow the steps below to create a [pipeline](./glossary.md/#pipeline) from the Twitter API to 
+Google BigQuery from scratch. The same steps can be repeated for any source and destination of your 
+choice--use `dlt init <source> <destination>` and then build the pipeline for that API instead.
+
+Please make sure you have [installed `dlt`](./installation.mdx) before getting started here.
+
+## 1. Initialize project
+
+Create a new empty directory for your `dlt` project by running
+```
+mkdir twitter-bigquery
```

Comment:
should we suggest to create virtual env? or just to refer to quickstart?

Reply:
We tell them to create a virtual env in installation: "Please make sure you have [installed `dlt`](./installation.mdx) before getting started here."